packages/validator/src/metrics.ts

```diff
@@ -325,6 +325,12 @@ export function getMetrics(register: MetricsRegister, gitData: LodestarGitData)
       help: `Current health status of the beacon(s) the validator is connected too. ${renderEnumNumeric(BeaconHealth)}`,
     }),
 
+    validatorStatuses: register.gauge<{status: string}>({
+      name: "vc_validator_status_count",
+      help: "Current statuses of the validators.",
```

Comment:
super nit picky but other metrics don't end with a period, let's improve consistency

```suggestion
      help: "Current statuses of the validators",
```